tests/test_blas_ops.py

```diff
@@ -404,3 +404,146 @@ def test_accuracy_addr(M, N, dtype):
         res_out = torch.addr(input_tensor, vec1, vec2, alpha=alpha, beta=beta)
 
     gems_assert_close(res_out, ref_out, dtype, equal_nan=True)
+
+
+def get_test_params():
```

Comment:
Please avoid using such a generic function name.
No one in the future can quickly realize that this is a helper function for the `test_scaled_mm` test case.